src/App.js

```diff
@@ -1,12 +1,87 @@
-import React from 'react';
-import './App.css';
+
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import PokemonCard from './components/PokemonCard/PokemonCard';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonList, setPokemonList] = useState([]);
+    const [nextLink, setNextLink] = useState('');
+    const [previousLink, setPreviousLink] = useState('');
```

Comment:
Deze had je samen kunnen voegen tot één:
```suggestion
    const [link, setLink] = useState("https://pokeapi.co/api/v2/pokemon/?limit=20");
```